### ISSUE TITLE
Regarding the OpenCompass Evaluation

### ISSUE THREAD

**Issue by small-noob** [Clin0212/HydraLoRA#21]
Hello, thank you for your work. I am a newcomer to this field, and I would like to ask about the direct reason for adding the following code in OpenCompass (is it due to the requirements of specific packages such as peft and transformers?), and whether there are better optimization methods.
```
import sys
sys.path.insert(0, 'path_to_your_current_dir_containing_changed_peft&transformers')
```
Additionally, I would like to inquire why, in the example you provided, the same data (from what I observed, data/example_data/train.json and test.json contain identical data) is used for both training and evaluation. Also, in the code you provided, the training set includes all the *.json files from data/example_data, which results in the training set containing the validation set.

---
Once again, thank you for your excellent work.

**Comment by Clin0212** [comment]
Thank you for your questions.

1. The reason for adding this code is that HydraLoRA modifies the source code of the LoRA Layer in [`peft`](https://github.com/Clin0212/HydraLoRA/blob/main/HydraLoRA/peft/tuners/lora.py). Therefore, OpenCompass needs to import the modified `peft` LoRA path. You can also use other evaluation codes, as long as the modified LoRA files are correctly referenced. You can also consider referring to the simpler and clearer [multi-modal version](https://github.com/Clin0212/HydraLoRA/blob/main/MLLM-HydraLoRA/moe.py).

2. Regarding the example data: the `example_data` folder is only meant as a toy example. The identical data in `train.json` and `test.json` is simply for demonstration purposes, and not intended for practical application.

**Comment by small-noob** [comment]
> Thank you for your questions.
> 
> 1. The reason for adding this code is that HydraLoRA modifies the source code of the LoRA Layer in [`peft`](https://github.com/Clin0212/HydraLoRA/blob/main/HydraLoRA/peft/tuners/lora.py). Therefore, OpenCompass needs to import the modified `peft` LoRA path. You can also use other evaluation codes, as long as the modified LoRA files are correctly referenced. You can also consider referring to the simpler and clearer [multi-modal version](https://github.com/Clin0212/HydraLoRA/blob/main/MLLM-HydraLoRA/moe.py).
> 2. Regarding the example data: the `example_data` folder is only meant as a toy example. The identical data in `train.json` and `test.json` is simply for demonstration purposes, and not intended for practical application.

Thank you for address my concerns!
